Frameworks/UIKit/UITextField.mm

```diff
@@ -93,18 +72,14 @@ @implementation UITextField {
     BOOL _adjustsFontSizeToFitWidth;
```

Comment:
removed bunch of them. and correct the annotations.

---
In reply to: [110058933](https://github.com/Microsoft/WinObjC/pull/2411#discussion_r110058933) [](ancestors = 110058933)